### ISSUE TITLE
Unable to activate the button with NVDA

### ISSUE THREAD

**Issue by Testerforwork** [nvaccess/nvda#13304]
![Button](https://user-images.githubusercontent.com/98935498/152286681-4d7545bd-51cd-44e3-bd62-cd1bfb39338e.png)
Hello NVDA Team:
I am unable to activate the button with NVDA using the Chrome browser. But I am able to activate it while using other browsers like Firefox with NVDA.

### Steps to reproduce:
Open the URL in Chrome browser - https://www.cslplasma.com/why-donate
Turn on the NVDA screen reader and try to activate the buttons - 'What is Blood Plasma?', 'What Does Blood Plasma Do?', 'How Does the Body Make Plasma?' using enter or space keys.

### Actual behavior:
Unable to activate\expand the buttons with NVDA.

### Expected behavior:
We should be able to activate\expand the buttons with NVDA.

### System configuration
Chrome Version:  98.0.4758.81 (Official Build) (64-bit)

#### NVDA version:
2021.3.1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Working fine with NVDA in Firefox browser(version - 96.0.3 (64-bit).
Working fine with JAWS(version 2021) in Chrome browser(version - 98.0.4758.81 (Official Build) (64-bit)) and Firefox browser(version - 96.0.3 (64-bit)

#### Does the issue still occur after restarting your computer?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Testerforwork** [comment]
I am able to activate the buttons only in forms mode/focus mode.

**Comment by Brian1Gaff** [comment]
I have come across this same thing with download buttons on some sites, it 
is very hit and miss though, and sometimes it affects all browsers, and 
sometimes not. Maybe its some odd authoring issue on the pages that needs a 
work around?

It is tied up with focus mode, so I guess you could set up the automatic 
focus mode so it will always go into that when the problem code occurs.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by ABuffEr** [comment]
> so I guess you could set up the automatic focus mode so it will always go into that when the problem code occurs.
Same tought, but it's not the case here...
It sureley depends from a page impelmentation aspect (with "Show options" on this page happens exactly the opposite, issues in Firefox, and none in Chrome), but I'm not sure that NVDA could notice when the button has worked or not, and so fix the problem.

**Comment by Testerforwork** [comment]
I am facing a similar issue now with the checkboxes (Option 1, Option 2, Option 3) 
Only in the Chrome browser. NVDA neither announces the role checkbox nor announces its name.
When I try the same with Focus mode, it works as expected.
![Uploading Checkbox.png…]()


Test URL: https://omniarc-gaming-mouse.myshopify.com/pages/deque-ada-audit-test-page

**Comment by Testerforwork** [comment]
> I have come across this same thing with download buttons on some sites, it is very hit and miss though, and sometimes it affects all browsers, and sometimes not. Maybe its some odd authoring issue on the pages that needs a work around? It is tied up with focus mode, so I guess you could set up the automatic focus mode so it will always go into that when the problem code occurs. Brian ***@***.*** Sent via blueyonder. Please address personal E-mail to:- ***@***.***, putting 'Brian Gaff' in the display name field. Newsgroup monitored: alt.comp.blind-users ----- Original Message ----- From: "Testerforwork" ***@***.***> To: "nvaccess/nvda" ***@***.***> Cc: "Subscribed" ***@***.***> Sent: Friday, February 04, 2022 6:54 AM Subject: Re: [nvaccess/nvda] Unable to activate the button with NVDA (Issue #13304)
> […](#)
> I am able to activate the buttons only in forms mode/focus mode. -- Reply to this email directly or view it on GitHub: [#13304 (comment)](https://github.com/nvaccess/nvda/issues/13304#issuecomment-1029706687) You are receiving this because you are subscribed to this thread. Message ID: ***@***.***>

NVDA screen reader should switch to focus mode predictably.
